### PR TITLE
Added missing Spanish translation

### PR DESCRIPTION
**Pull request by fcosueza**
I've added some missing fields in spanish translation or the app. :+1:

### CODE REVIEW

**Comment by dodie** [comment]
Thank you! :beers:

**Comment by fcosueza** [comment]
Hey, on line 1 of my changes i introduce unintentionally a " : ". I just realize that, sorry. This may cause problems in the first element of the json.

File: i18n/translations/es.json [hunk 1]
```diff
@@ -1,12 +1,22 @@
-{
+{:
```
Comment (fcosueza) [inline i18n/translations/es.json]: That " : " must be deleted, I put it unintentionally.

**Comment by dodie** [comment]
Thanks, fixed

**Pull request merged**